performance/gatling_tests/src/gatling/resources/data/nodeJSAction.js

```diff
@@ -0,0 +1,5 @@
+function main(params) {
+    var greeting = "Hello" + (params.text || "stranger") + "!";
+    console.log(greeting);
+    return { payload: greeting };
+}
```

Comment:
It seems unnecessary to duplicate these tests aren’t there a consistent set of basic actions for all runtimes in the tests/dat/actions folder?

Reply:
I checked in these actions here to ensure, that all actions do exactly the same. By ensuring this, we can also compare the runtimes against each others.

In the `tests/dat/actions` are a lot of actions, but I didn't find one action, that is doing exactly the same in all languages.

Reply:
Thanks for checking.